anime_downloader/sites/animesuge.py

```diff
@@ -35,6 +34,81 @@ def _scrape_metadata(self):
 
 
 class AnimeSugeEpisode(AnimeEpisode, sitename='animesuge'):
+    """
+    Decodes the encoded string returned by /ajax/anime/episodes to the embed url.
+
+    Python implementation of decode9AnimeURL, it's basically identical to the js one
+    """ 
+    def decodeString(self, str):
```

Comment:
If both are the same code, maybe move it util or something?

Reply:
Made AnimeSugeEpisode inherit NineAnimeEpisode and got rid of the duplicate method